models/intel/person-detection-0201/accuracy-check.yml

```diff
@@ -0,0 +1,19 @@
+models:
+  - name: person-detection-0201
+
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: ssd
+          labels_shift: 1
```

Comment:
if it is need only for shift labels to align with annotation (0 - backgroud label, 1-... valid labels), then you can just convert dataset with has_backgroud: False and it should work without prediction labels shift

Reply:
ok, fixed